packages/components/src/draggable/stories/index.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import type { DragEvent } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import { Icon, more } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import Draggable from '..';
+
+const meta: ComponentMeta< typeof Draggable > = {
+	component: Draggable,
+	title: 'Components/Draggable',
+	argTypes: {
+		elementId: { control: { type: null } },
+	},
+	parameters: {
+		actions: { argTypesRegex: '^on.*' },
+		controls: { expanded: true },
+		docs: { source: { state: 'open' } },
```

Comment:
I guess we should hide the source for this one too since it isn't very useful?

Reply:
The example should work now, so I think we could hide the source 👍  Will update later today.